.github/workflows/test_coverage.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         gcloud auth activate-service-account --key-file metrics-access.json
     - name: Build code coverage tool
       run: |
-        cd firebase-ios-sdk/scripts/code_coverage_report/generate_code_coverage_report/
+        cd firebase-ios-sdk/scripts/health_metrics/generate_code_coverage_report/
```

Comment:
this can be one line with `--package-path` option:
`swift build --package-path firebase-ios-sdk/scripts/health_metrics/generate_code_coverage_report/`